services/core-web/src/tests/components/modalContent/ReplaceDocumentModal.spec.tsx

```diff
@@ -7,7 +7,7 @@ import { MINEDOCUMENTS } from "@mds/common/tests/mocks/dataMocks";
 import FormWrapper from "@mds/common/components/forms/FormWrapper";
 import { FORM } from "@mds/common/constants/forms";
 
-const props = {
+const props: any = {
```

Comment:
When I expand the test here- I'm thinking that the FormWrapper should _not_ be included as part of the test, since the component should handle the reduxForm on its own.